frontend/src/router.ts

```diff
@@ -12,6 +13,7 @@ export const ROUTES = {
   ORDER: { name: "order", path: "/order", label: "Aufträge" },
   DOCUMENTS: { name: "documents", path: "/documents", label: "Dokumente" },
   ARTICLES: { name: "articles", path: "/articles", label: "Artikel" },
+  OFFER: { name: "offer", path: "/offer", label: "Offer" },
 };
```

Comment:
I wonder if the current `ROUTES` object still makes sense.

With all the sub-paths we already have and the manual concatenation I don't see the routes object as a help anymore. We don't use it for the full paths anyway and even the labels of sub-paths are not part of this  object.

The main benefit I see is the "label" but we can move that to the navigation component.

What do you think?